org.mixedrealitytoolkit.input/InteractionModes/NearInteractionModeDetector.cs

```diff
@@ -16,63 +16,106 @@ namespace MixedReality.Toolkit.Input
     [AddComponentMenu("MRTK/Input/Near Interaction Mode Detector")]
     public class NearInteractionModeDetector : ProximityDetector
     {
+        /// <summary>
+        /// The set of near interactables that belongs to near interaction
+        /// </summary>
         [SerializeField]
-        [Tooltip("The set of near interactors that belongs to near interaction")]
-        private List<XRBaseInteractor> nearInteractors;
+        [Tooltip("The set of near interactables that belongs to near interaction")]
+        private List<XRBaseInteractable> nearInteractables;
```

Comment:
Please let me know if I missed a comment thread here for context, but I'm not sure why this was changed from interactors to interactables. This would be a breaking change, and also one that makes this component very difficult to keep up to date.

Previously, while storing a serialized list of interactors, it's easy to ensure all near interactors for each hand are represented here, because there aren't many (our current default is two per-hand):

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/3580640/65a224ae-4030-4962-9dc0-167b7a12979f)

With this change, does that mean I need to serialize every single near interactable in every single scene I might be using? That feels quickly unwieldy. Where is this currently being filled?

It also prevents us from keeping the MRTK XR Rig prefab in a usable state out-of-the-box, as a developer will need to assign every interactable in the Inspector before use.

I strongly feel this should be reverted.

Reply:
Ack I missed this!   Yesh we should not be changing that..

Reply:
I suspect this was a Find and Replace error

Reply:
Reverted in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/ee2f619a385491f73d7303e544ff944704e0042f, resolving.